tests/python_package_test/test_plotting.py

```diff
@@ -131,16 +132,14 @@ def test_create_tree_digraph(self):
         self.assertEqual(len(graph.graph_attr), 0)
         self.assertEqual(len(graph.edge_attr), 0)
         graph_body = ''.join(graph.body)
-        self.assertIn('threshold', graph_body)
-        self.assertIn('split_feature_name', graph_body)
-        self.assertNotIn('split_feature_index', graph_body)
-        self.assertIn('leaf_index', graph_body)
-        self.assertIn('split_gain', graph_body)
-        self.assertIn('internal_value', graph_body)
-        self.assertIn('internal_weight', graph_body)
-        self.assertNotIn('internal_count', graph_body)
-        self.assertNotIn('leaf_count', graph_body)
-        self.assertNotIn('leaf_weight', graph_body)
+        self.assertIn('leaf', graph_body)
```

Comment:
Add
```
self.assertIn('feature', graph_body)
```
as well.

Reply:
Unfortunately that wouldn't work as is. As you asked in a previous comment, I added the word "feature" for clarity, but just in the case where `feature_names` do not exist.
```
if feature_names is not None:
    label = '<B>{0}</B> {1} '.format(feature_names[root['split_feature']], operator)
else:
    label = 'feature <B>{0}</B> {1} '.format(root['split_feature'], operator)
```

In the case `feature_names` do exist it can be seen as superfluous either because the user specified the name itself so they should know what it is, either because default names are "Column_x", x being a number. In the test, names are not mentioned, but `feature_names` is not None (being and contains names of the form "Column_x"), so the word "feature" does not appear in the text. 

I can either add the word "feature" even when `feature_names` are not None, or forget the assertion. Let me know what is best.

Reply:
Sorry, my bad! Everything is OK with assertions.